ocaml/xcp-rrdd/bin/rrdd/rrdd_server.ml

```diff
@@ -112,11 +112,11 @@ let backup_rrds (remote_address : string option) () : unit =
   let destination =
     match remote_address with
     | None ->
-        "to local disk"
+        "local disk"
     | Some address ->
-        Printf.sprintf "to host %s" address
+        Printf.sprintf "host %s" address
   in
-  debug "backing up rrds %s" destination ;
+  info "%s: trying to back up RRDs to %s" __FUNCTION__ destination ;
```

Comment:
How often does this happen?

Reply:
every hour during normal operation, once every 6 minutes when testing, plus any manual requests to back the RRDs, if any

Reply:
This includes saving to local disk? RRDs are just in memory for most of the hour?